lib/controllers/root-controller.js

```diff
@@ -488,8 +497,30 @@ export default class RootController extends React.Component {
 
   async viewChangesForCurrentFile(stagingStatus) {
     const editor = this.props.workspace.getActiveTextEditor();
-    const absFilePath = editor.getPath();
+    const absFilePath = await realPath(editor.getPath());
     const repoPath = this.props.repository.getWorkingDirectoryPath();
+    if (repoPath === null) {
+      const [projectPath] = this.props.project.relativizePath(editor.getPath());
+      const notification = this.props.notificationManager.addInfo(
+        "Hmm, we don't have anything to compare this to.",
```

Comment:
@jasonrudolph Material Design has lots of guidelines around writing messages in products. Here the section about "you" and "we": https://material.io/guidelines/style/writing.html#writing-language

tl;dr: Avoid the pronoun “we”.